internal/venice-common/src/main/java/com/linkedin/venice/pubsub/manager/TopicManager.java

```diff
@@ -714,7 +714,7 @@ public void close() {
       logger.warn("{} is already closed", this);
     }
     try {
-      closeAsync().get(600, TimeUnit.SECONDS);
+      closeAsync().get(2, TimeUnit.MINUTES);
```

Comment:
That is a long time... why wait so long? What if we crash and the close doesn't get executed at all, would it be a problem? I don't think there is anything in TopicManager that must absolutely complete at all costs before shutting down, is there?

Reply:
>I don't think there is anything in TopicManager that must absolutely complete at all costs before shutting down

Yeah, I think you're correct. 


I considered this as a defense against unbounded blocking during close, and since it's not expected to occur frequently I was thinking maybe 2min is ok.   

Should we consider reducing it to perhaps 30 seconds?

Reply:
Removed blocking wait on close